src/less/documentation-swedbankpay.less

```diff
@@ -63,6 +75,12 @@
                 }
             }
         }
+
+        --preview-min-height: 200px;
```

Comment:
Just curious, the double dash (--) is it custom for something?

Reply:
this is a CSS variable